ogusa/aggregates.py

```diff
@@ -90,12 +90,12 @@ def get_I(b_splus1, K_p1, K, p, method):
                            (1, 1, p.J))).sum(1).sum(1)) /
                  (1 + np.squeeze(np.hstack((p.g_n[1:p.T], p.g_n_ss)))))
         aggI = ((1 + np.squeeze(np.hstack((p.g_n[1:p.T], p.g_n_ss)))) *
-                np.exp(p.g_y) * (K_p1 - part2) - (1.0 - p.delta) * K)
+                np.exp(p.g_y) * (B_p1 - part2) - (1.0 - p.delta) * B)
 
     return aggI
 
 
-def get_K(b, p, method, preTP):
+def get_B(b, p, method, preTP):
```

Comment:
There are a couple instances of `K` needing to be changed to `B` in this docstring.